src/core/dbus-manager.c

```diff
@@ -3035,6 +3036,9 @@ static int method_start_aux_scope(sd_bus_message *message, void *userdata, sd_bu
         if (r == 0)
                 return 1; /* No authorization for now, but the async polkit stuff will call us again when it has it */
 
+        log_once(LOG_WARNING, "StartAuxiliaryScope() is deprecated because state of resources cannot be migrated between cgroups. "
+                              "Please report this to systemd-devel@lists.freedesktop.org or "
+                              "https://github.com/systemd/systemd/issues/ if you see what unit uses this");
```

Comment:
"… if you see this message and know the software making use of this functionality."

Reply:
could you also add the SD_BUS_VTABLE_DEPRECATED flag to the bus call in the vtable?